src/SignalR/server/SignalR/test/HubConnectionHandlerTests.cs

```diff
@@ -859,10 +859,17 @@ public async Task HubMethodsAreCaseInsensitive(Type hubType)
         [InlineData(nameof(MethodHub.MethodThatYieldsFailedTask), false)]
         public async Task HubMethodCanThrowOrYieldFailedTask(string methodName, bool detailedErrors)
         {
+            var hasErrorLog = false;
             bool ExpectedErrors(WriteContext writeContext)
             {
-                return writeContext.LoggerName == "Microsoft.AspNetCore.SignalR.Internal.DefaultHubDispatcher" &&
+                var expected = writeContext.LoggerName == "Microsoft.AspNetCore.SignalR.Internal.DefaultHubDispatcher" &&
```

Comment:
`String.Equals()` ?

Reply:
Yeah, maybe I'll do a pass of all our tests later and do that